src/Modules/ModParser.lua

```diff
@@ -498,6 +498,7 @@ local modNameList = {
 	["physical damage over time multiplier"] = "PhysicalDotMultiplier",
 	["fire damage over time multiplier"] = "FireDotMultiplier",
 	["cold damage over time multiplier"] = "ColdDotMultiplier",
+	["cold damage over time"] = { "ColdDamage", keywordFlags = KeywordFlag.ColdDot},
```

Comment:
This mod should go a couple of lines higher, where the other "x damage over time" mods are defined.

Reply:
Done!